packages/app-btc/src/operations/signTxn/index.ts

```diff
@@ -113,15 +118,13 @@ export const signTxn = async (
         ),
         value: output.value,
         isChange: output.isChange,
-        chainIndex: output.chainIndex,
-        addressIndex: output.addressIndex,
+        changesIndex: output.addressIndex,
```

Comment:
```suggestion
        changeIndex: output.addressIndex,
```

Reply:
not possible ig. will first need to change in proto. then here.